src/main/java/mate/academy/Main.java

```diff
@@ -44,7 +57,8 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService
```

Comment:
The variable declaration for `MovieSessionService` is missing an `=` sign. It should be `MovieSessionService movieSessionService = ...`.